### PR TITLE
Sample code for the article on string formatting tools

### PR DESCRIPTION
**Pull request by lpozo**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: python-formatted-output/format.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+print(f"{3.14159:g}")
+print(f"{-123456789.8765:g}")
+print("{:g}".format(3.14159))
+print("{:g}".format(-123456789.8765))
+
+print(f"{-123456789.8765:G}")
+print("{:G}".format(-123456789.8765))
+
+infinite = 1e300 * 1e300
```
Comment (gahjelle) [inline python-formatted-output/format.py:15]: We can change this to `float("inf")` as well :)

File: python-formatted-output/interpolation.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# print(f"Single-line f-string with single quotes")
+# print(f"Single-line f-string with double quotes")
+# print(
+#     f"""Multiline triple-quoted f-string
+# with single quotes"""
+# )
+# print(
+#     f"""Multiline triple-quoted f-string
+# with double quotes"""
+# )
```
Comment (gahjelle) [inline python-formatted-output/interpolation.py:13]: I think Black has already messed up these examples (before the comments where added?). So we should redo the f vs F and single vs double quotes.

File: python-formatted-output/person.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    def __repr__(self):
+        return f"{type(self).__name__}(name='{self.name}', age={self.age})"
+
+
+jane = Person("Jane", 25)
+f"{jane!s}"
+f"{jane!r}"
+"{jane!s}".format(jane=jane)
+"{jane!r}".format(jane=jane)
```
Comment (gahjelle) [inline python-formatted-output/person.py:17]: Should we add `print()` calls on these to make them show up when running as a script?

**Review by gahjelle — COMMENTED** [review]
@lpozo This looks great! I only have a few small comments on things we can fix up a bit.

**Comment by lpozo** [comment]
@gahjelle Hey! I did the updates. Thanks for your feedback.

**Review by gahjelle — APPROVED** [review]
@lpozo Looks great! :+1:

**Review by brendaweles — APPROVED** [review]

**Pull request merged**